mage_integrations/mage_integrations/destinations/oracledb/__init__.py

```diff
@@ -178,7 +180,11 @@ def build_insert_commands(
                     # Build update command
                     insert_value_without_left_parenthesis = re.sub(r"^\(", "", insert_value)
                     insert_value_strip = re.sub(r"\)$", "", insert_value_without_left_parenthesis)
-                    updated_values = insert_value_strip.split(',')
+                    insert_value_strip = re.sub(r"\,$", "", insert_value_strip)
+                    splitted_values = insert_value_strip.split('),')
+                    updated_values = [s + ")" if i < len(splitted_values) - 1 else s
```

Comment:
what's this code for?

Reply:
I added a comment with example to explain.

Basically we want to convert `insert_value` from the format of `CAST('1' AS NUMBER), TO_DATE('2023-09-10', 'yyyy-mm-dd'), to_nclob('{\"customer\": \"John Doe\", \"items\": {\"product\": \"Beer\", \"qty\": 6}}'),` 

to

an array of each value with 
equal sign ["col1 = CAST('1' AS NUMBER)", "col2 =  TO_DATE('2023-09-10', 'yyyy-mm-dd')", 'col3 =  to_nclob(\'{"customer": "John Doe", "items": {"product": "Beer", "qty": 6}}\')']

In order to do that, I basically
- First remove unnecessary punctuation (1) ( at beginning (2) ) at the end (3), at the end
- split by "),"
- Add back ) at the end of each value